src/VariableSizeGrid.js

```diff
@@ -19,7 +19,7 @@ type ItemMetadata = {|
   offset: number,
   size: number,
 |};
-type ItemMetadataMap = { [index: number]: ItemMetadata };
+type ItemMetadataMap = { [index: number]: ItemMetadata, ... };
```

Comment:
I'm unfamiliar with this "..." syntax, and my attempts to find it in the docs are failing. Pointers?

Reply:
Looks like CI is unfamiliar with it also 😆 

![Screen Shot 2019-06-11 at 8 27 11 AM](https://user-images.githubusercontent.com/29597/59285367-bda51480-8c22-11e9-8908-8caa5d9770ae.png)

Reply:
Explicit inexact object. Part of transition to exact by default.
https://github.com/facebook/flow/releases/tag/v0.101.0

Reply:
Will add it later then.

Reply:
Gotcha. Thanks. I don't think they've published the 101 notes internally yet. I've only read v100.